crates/api_crud/src/post/update.rs

```diff
@@ -62,14 +57,6 @@ impl PerformCrud for EditPost {
     let post_id = data.post_id;
     let orig_post = Post::read(context.pool(), post_id).await?;
 
-    check_community_ban(
-      local_user_view.person.id,
-      orig_post.community_id,
-      context.pool(),
-    )
-    .await?;
```

Comment:
I think this check should stay to prevent banned users from editing their posts.

Reply:
Done, added them back in.